source/extensions/filters/http/jwt_authn/filter_config.cc

```diff
@@ -55,7 +56,7 @@ FilterConfigImpl::FilterConfigImpl(
 
   if (proto_config_.has_filter_state_rules()) {
     filter_state_name_ = proto_config_.filter_state_rules().name();
-    for (const auto& it : proto_config_.filter_state_rules().requires()) {
+    for (const auto& it : proto_config_.filter_state_rules().requires_()) {
```

Comment:
`requires_()` is a bit curious. Is this some sort of proto v23 naming change to avoid collisions or some such?

Reply:
`requires` is a C++20 keyword. If it is used to name a proto field, the proto compiler uses `requires_` in generated C++ classes. PGV needs to have the same change otherwise it tried to validate wrong filed name.

Reply:
Ah! That makes sense. Thanks!